lib/Conversion/ImportVerilog/Structure.cpp

```diff
@@ -106,6 +224,9 @@ struct MemberVisitor {
     if (!lhs || !rhs)
       return failure();
 
+    if (lhs.getType() != rhs.getType())
```

Comment:
If not, there is an error: `logic[3:2]` and `logic[1:0]` mismatch.

Reply:
Same as above: can you PR this directly upstream, with a test that checks for this exact problem (`3:2` vs `1:0`)?

Reply:
I'll push this together with the above-mentioned.

Reply:
`3:2` vs `1:0` exactly exists, but I think maybe there is a wrong mark at this position. This comment is outdated, and I forget why I left a comment here.